src/audio/kpb.c

```diff
@@ -45,12 +45,28 @@
 #include <sof/audio/buffer.h>
 #include <sof/ut.h>
 
+/*! KPB private data */
+struct comp_data {
+	/* runtime data */
+	uint8_t no_of_clients; /**< number of registered clients */
```

Comment:
Best to make these a uint32_t since it generates better xtensa code.

Reply:
@lgirdwood this is outdated code, has been changed already.

Reply:
ok, but can this be removed from patches since it appears in the GH PR (and will be merged in the history). Best to squash it at your end to remove it.